tools/openms/macros.xml

```diff
@@ -4,7 +4,7 @@
      add additional macro files using the m/macros parameter -->
 <macros>
   <token name="@TOOL_VERSION@">2.5</token>
-  <token name="@GALAXY_VERSION@">0</token>
+  <token name="@GALAXY_VERSION@">1</token>
```

Comment:
do we want to increase it or patch the older version?

Reply:
whatever you think is better.